v2/product/product-type.rst

```diff
@@ -1,4 +1,68 @@
 Create a product type
 =====================
+For this example we will create a product type for T-shirts
+## First create Product attribute
```

Comment:
A good heading here would be "Product Attributes"

Be sure to link to the Product Attributes section for more information. Let the person reading this documentation know why you've chosen to create attributes. They will want to understand your decision so they can use that to make decisions for their own data model. 

For example: 

> Not all products have attributes. For our example product type, we're creating shirts. Shirts usually come in colors and sizes. One type of shirt may have a few sizes that it comes in. Those sizes are what we call attributes. And those attributes can be created like this: